sdk/python/tests/cli/test_cli_local.py

```diff
@@ -1,3 +1,54 @@
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+from textwrap import dedent
+from typing import List
+
+from feast import cli
+
+
+class CliRunner:
+    """
+    NB. We can't use test runner helper from click here, since it doesn't start a new Python
+    interpreter. And we need a new interpreter for each test since we dynamically import
+    modules from the feature repo, and it is hard to clean up that state otherwise.
+    """
+
+    def run(self, args: List[str], cwd: Path) -> subprocess.CompletedProcess:
```

Comment:
Do we need a class + method, or can this be a function?

Reply:
My plan was to make this class _also_ set up a temporary repo directory and maybe include some other helpers but i didn't get around to it yet